cli/npm/resolution/graph.rs

```diff
@@ -552,18 +551,15 @@ impl<'a, TNpmRegistryApi: NpmRegistryApi>
 
   fn resolve_node_from_info(
     &mut self,
-    name: &str,
+    pkg_req_name: &str,
     version_matcher: &impl NpmVersionMatcher,
     package_info: &NpmPackageInfo,
     parent_id: Option<&NpmPackageId>,
   ) -> Result<Arc<Mutex<Node>>, AnyError> {
-    let version_and_info = self.resolve_best_package_version_and_info(
-      name,
-      version_matcher,
-      package_info,
-    )?;
+    let version_and_info = self
+      .resolve_best_package_version_and_info(version_matcher, package_info)?;
     let id = NpmPackageId {
-      name: name.to_string(),
+      name: package_info.name.to_string(),
```

Comment:
I think this was also incorrect. The package id name should be the resolved name and not the key in the npm package dependency entry.